docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Configuration
+sidebar_order: 6100
+description: "Learn about the User Feedback Widget configuration options."
+---
+
+The User Feedback Widget offers many customization options, and if the available options are insufficient, you can [use your own UI](/platforms/react-native/user-feedback/#user-feedback-api).
+
+![An image showing the main customization options for the User Feedback Widget](./img/mobile-user-feedback-widget-customization.png)
+To collect user feedback from inside your application use the `showFeedbackWidget` method.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
```

Comment:
Let's also hight light that the Root component has to be wrapped with `Sentry.wrap` for this to work.

Reply:
Good point 👍 
Added with 8a84abf914211305d602c8663e43be43de4d5b91